helm/prometheus-agent/values.yaml

```diff
@@ -12,6 +12,9 @@ global:
 
   externalLabels: {}
 
+image:
+  tag: ""
```

Comment:
oh, that's a nice one: adding a template to the parent chart required to interpret _helpers, which required this key to exist.
As I since moved the template to the subchart, I don't need this one anymore.